iFixit/checkout-ui-extensions
# Add community links to order status and thank you pages

Closes https://github.com/iFixit/checkout-ui-extensions/issues/18

We now show a community link on `Thank You` and `Order Status` pages. This is a dynamic component, so testing requires us to specify where we want to place it.

![image](https://github.com/iFixit/checkout-ui-extensions/assets/95708883/db433d56-7827-4941-8210-c2195f4a6649)


QA
--
1. Run `pnpm install`, then run `pnpm dev`
2. Select the `purchase.thank-you.block.render` target link.
3. Append `?placement-reference=ORDER_STATUS3` to the URL at checkout
4. Complete checkout
5. At the `Thank you` page (right after placing order), ensure you can see the component
6. Refresh the page and observe that the url changes to `order status` from `thank you`
7. Observe that you can still see the component

*Note on step 6: I was unable to refresh the page to get to the `order status` page. If you can't get this to work, you can also verify that the component does render on order status by following the steps below:
1. Click the `App Home` link in the preview console
2. Click `Online Store` in the left menu 
3. On the current theme, click `Customize`
4. In the top middle-right dropdown, click `Checkout`
5. Click the dropdown in the top middle and click `Order Status`
6. You should be able to see the component on the order status page.
